src/JBSplits.sol

```diff
@@ -172,9 +170,6 @@ contract JBSplits is JBControlled, IJBSplits {
             // The percent should be greater than 0.
             if (splits[i].percent == 0) revert INVALID_SPLIT_PERCENT();
 
-            // `projectId` should fit within a uint56
```

Comment:
Why is this removed? setSplitGroupsOf in controller still accepts a uint256 projectId

Reply:
the splits.projectId is guaranteed within a uint56 because the type is such. This comment and the line underneath are not necessary.

Reply:
<img width="899" alt="image" src="https://github.com/user-attachments/assets/c672e37b-cf5a-4664-98e0-8b770fa1340e">

the projectId that is 256 is the one that stores the mapping. the projectId in the split, which is being checked here, is uint56.